[2023-08-14T16:19:34.131+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: emrPipeline.validate_file manual__2023-08-14T16:18:38+00:00 [queued]>
[2023-08-14T16:19:34.251+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: emrPipeline.validate_file manual__2023-08-14T16:18:38+00:00 [queued]>
[2023-08-14T16:19:34.252+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-14T16:19:34.421+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): validate_file> on 2023-08-14 16:18:38+00:00
[2023-08-14T16:19:34.500+0000] {standard_task_runner.py:57} INFO - Started process 9982 to run task
[2023-08-14T16:19:34.607+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'emrPipeline', 'validate_file', 'manual__2023-08-14T16:18:38+00:00', '--job-id', '1471', '--raw', '--subdir', 'DAGS_FOLDER/emr.py', '--cfg-path', '/tmp/tmpmkl8qw4k']
[2023-08-14T16:19:34.675+0000] {standard_task_runner.py:85} INFO - Job 1471: Subtask validate_file
[2023-08-14T16:19:34.940+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-08-14T16:19:35.437+0000] {task_command.py:410} INFO - Running <TaskInstance: emrPipeline.validate_file manual__2023-08-14T16:18:38+00:00 [running]> on host d4028c5ba2ac
[2023-08-14T16:19:36.021+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='emrPipeline' AIRFLOW_CTX_TASK_ID='validate_file' AIRFLOW_CTX_EXECUTION_DATE='2023-08-14T16:18:38+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-14T16:18:38+00:00'
[2023-08-14T16:19:36.037+0000] {logging_mixin.py:150} INFO - Reading the file
[2023-08-14T16:19:36.151+0000] {base.py:73} INFO - Using connection ID 'aws_connection' for task execution.
[2023-08-14T16:19:36.152+0000] {logging_mixin.py:150} INFO - {"region_name": "us-east-1"}
[2023-08-14T16:19:36.231+0000] {credentials.py:1123} INFO - Found credentials in environment variables.
[2023-08-14T16:19:37.114+0000] {logging_mixin.py:150} INFO - VendorID,lpep_pickup_datetime,lpep_dropoff_datetime,store_and_fwd_flag,RatecodeID,PULocationID,DOLocationID,passenger_count,trip_distance,fare_amount,extra,mta_tax,tip_amount,tolls_amount,ehail_fee,improvement_surcharge,total_amount,payment_type,trip_type
[2023-08-14T16:19:37.176+0000] {python.py:183} INFO - Done. Returned value was: False
[2023-08-14T16:19:37.523+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=emrPipeline, task_id=validate_file, execution_date=20230814T161838, start_date=20230814T161934, end_date=20230814T161937
[2023-08-14T16:19:37.722+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-14T16:19:37.867+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
